Parallel-Tests.ps1

```diff
@@ -1,10 +1,11 @@
 param(
     [string[]] $directories,
     [string] $testFilter,
-    [string] $outDir,
     [string] $dotnet)
 
-$maxDegreeOfParallelism = 4
```

Comment:
This was intentionally constrained because we saw issues with too many tests trying to execute in parallel. There's a chance that issue is gone now, but I doubt it.

Reply:
I was actually trying to accommodate environments with less that 4 (v-)cores available, which were causing the same problems that you probably originally encountered.
I can change this to Max(num_procs, 4) if preferred.

Reply:
Fixed: Limit parallel test jobs to max 4, or NUMBER_OF_PROCESSORS if that is less.